server/routes/api.js

```diff
@@ -269,12 +269,12 @@ router.put('/templates/:id', (req, res) => {
   });
 });
 
-router.put('/dashboards/:id', (req, res) => {
+router.put('/dashboards/:id?', (req, res) => {
   let { id } = req.params || {};
   let { script } = req.body || {};
 
   if (!id || !script) {
-    return res.end({ error: 'No id or script were supplied for the new dashboard' });
+    return res.json({ errors: {message: 'No id or script were supplied for the new dashboard', type: 'id'}} );
```

Comment:
Is this supposed to be error or errors? It should either be "error" or turned into an array